platform/engineering/internal-tools.md

```diff
@@ -138,68 +138,87 @@ If you don't already have an SSH public key, or you're not sure if you do, compl
+    ```
+    ❯ ls -al | grep .ssh
+    -rw-------    1 jeremybritt  staff      28 Jun 12 14:13 .lesshst
+    drwx------    8 jeremybritt  staff     256 Aug  6 11:27 .ssh
+    ```
+    * In this subdirectory you will have two files, `id_rsa_vagov` and `id_rsa_vagov.pub`
+    * To get the contents of your new public key (which will need to be provided in a future step), you can run `cat ~/.ssh/id_rsa_vagov.pub`. This should give a long string of random characters (e.g. ssh-rsa AAAAAjfje983jJL3j2....).
+    * \***PLEASE NOTE**\*: Any time your terminal is restarted, you will want to ensure your key is added by running `ssh-add -l`. If your key signature is not returned, or you receive output referring to your agent, run `eval $(ssh-agent -s)` and `ssh-add ~/.ssh/id_rsa_vagov`. There are also [instructions for Mac users](https://apple.stackexchange.com/questions/48502/how-can-i-permanently-add-my-ssh-private-key-to-keychain-so-it-is-automatically) to load their desired key(s) automatically.
     * Got it? Return to [Additional orientation steps for developers](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/orientation/request-access-to-tools.md#authorizekeys) and request that your SSH keys be authorized.
 
 ## Configure the SOCKS proxy
 
-These steps assume your SSH keys have been authorized and that you're running on Linux or OSX. There are slightly different commands to connect to the proxy depending on whether you are connected to the VA network or not. You will need to run the SOCKS proxy on your local system whenever you need access to tools on the `vetsgov-internal` domain.
+These steps assume your SSH keys have been authorized and that you're running on Linux, OSX, or have a Unix shell emulator like Git Bash (Windows). There are slightly different commands to connect to the proxy depending on whether you are connected to the VA network or not. You will need to run the SOCKS proxy on your local system whenever you need access to tools on the `*.vfs.va.gov` domain.
 
 1. Save the SSH configuration that you'll need locally to access the remote SSH servers.
-    * Right click on [this link](https://github.com/department-of-veterans-affairs/devops/raw/master/ssh/config) and save as: `~/.ssh/config_va`
-    * Edit `~/.ssh/config` and add this line at the top:
-    `Include ~/.ssh/config_va`
-
-1. Add your SSH key to your local agent with `ssh-add -K ~/.ssh/id_rsa_vetsgov` (for Windows, the command will not require the `-K` flag).
-
-1. Proceed to the instructions below to access the SOCKS proxy.
+    * Click [this link](https://github.com/department-of-veterans-affairs/devops/raw/master/ssh/config) and copy the entire URL within the new tab.
```

Comment:
```suggestion
    * Click <a href="https://github.com/department-of-veterans-affairs/devops/raw/master/ssh/config" target="_blank">this link</a> and copy the entire URL of the new tab.
```
maybe help ensure link opens in new tab?

Reply:
applied